### PR TITLE
Remove updates and promotions checkbox and dropdown

### PR DESCRIPTION
**Pull request by benstrumeyer**
* [x] Have you followed the guidelines in [CONTRIBUTING.md](https://github.com/ghostery/ghostery-extension/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you added an explanation of what your changes do?
* [x] Does your submission pass tests?
* [x] Did you lint your code prior to submission?

- Removes the updates and promotions checkbox on Step 1 of the Create Account View
- Automatically subscribe users to emails upon registering in the hub instead
- Remove the "We take your privacy very seriously and learn more" dropdown and display the FAQ on the page at all times

### CODE REVIEW

File: app/Account/AccountConstants.js [hunk 1]
```diff
@@ -38,4 +38,4 @@ export const GET_USER_SUBSCRIPTION_DATA_FAIL = 'GET_USER_SUBSCRIPTION_DATA_FAIL'
 export const GET_USER_SUBSCRIPTION_DATA_SUCCESS = 'GET_USER_SUBSCRIPTION_DATA_SUCCESS';
 
 // Opt into/out of updates and promotions sendgrid emails
-export const ACCOUNT_DATA_EMAIL_PREFERENCES_CHECKBOX_CHANGE = 'ACCOUNT_DATA_EMAIL_PREFERENCES_CHECKBOX_CHANGE';
```
Comment (wlycdgr) [inline app/Account/AccountConstants.js:41]: Same deal, just double checking this isn't being used anywhere else anymore. If not, and the user can no longer change their email preferences in the extension panel or either of the hubs, we should be able to dispense with the `SUBSCRIBE_TO_EMAIL_LIST` action also, no?

File: app/Account/AccountReducer.js [hunk 2]
```diff
@@ -119,11 +119,11 @@ export default (state = initialState, action) => {
...
 			};
 		}
-		case ACCOUNT_DATA_EMAIL_PREFERENCES_CHECKBOX_CHANGE: {
-			const { name, checked } = action.payload;
+		case SUBSCRIBE_TO_EMAIL_LIST: {
+			const { name } = action.payload;
 			let emailPreferences;
 			if (name === 'global') {
-				emailPreferences = { ...state.user.emailPreferences, ...{ global: checked } };
+				emailPreferences = { ...state.user.emailPreferences, ...{ global: true } };
```
Comment (wlycdgr) [inline app/Account/AccountReducer.js:126]: As alluded to above, do we need this action at all given that `notify_promotions` and `notify_upgrade_updates` now default to `true` in `ConfData`?

File: app/ghostery-browser-hub/Views/OnboardingViews/Step1_CreateAccountForm/Step1_CreateAccountFormContainer.jsx [hunk 3]
```diff
@@ -164,7 +156,7 @@ class CreateAccountFormContainer extends Component {
 			if (success) {
 				// User is automatically logged in, and redirected to the logged in view of BrowserCreateAccountForm
 				actions.getUser().then(() => {
-					if (isUpdatesChecked) actions.handleEmailPreferencesCheckboxChange('global', isUpdatesChecked);
+					actions.subscribeToEmailList('global');
```
Comment (wlycdgr) [inline app/ghostery-browser-hub/Views/OnboardingViews/Step1_CreateAccountForm/Step1_CreateAccountFormContainer.jsx:159]: See above - we may not need this at all?

File: app/ghostery-browser-hub/Views/OnboardingViews/Step1_CreateAccountView/Step1_CreateAccountView.jsx [hunk 4]
```diff
@@ -58,24 +57,8 @@ const Step1_CreateAccountView = (props) => {
...
-	const faqRef = useRef(null);
-	const scrollToFAQ = () => {
-		faqRef.current.scrollIntoView({ behavior: 'smooth' });
-	};
-
-	const handleFAQLearnMoreClick = () => {
-		setTimeout(scrollToFAQ, 1);
-		setExpanded(!expanded);
-	};
-
```
Comment (wlycdgr) [inline app/ghostery-browser-hub/Views/OnboardingViews/Step1_CreateAccountView/Step1_CreateAccountView.jsx:78]: I THINK we are only cutting the bit of copy above "Learn More" - not the click-arrow-to-expand mechanism. If that's so, we'll need to restore this logic and the other related bits

**Review by wlycdgr — COMMENTED** [review]
May need to restore Learn More expandable drawer logic depending on final UX decision on that. If that DID get cut, then ignore that comment!

File: app/Account/AccountActions.js [hunk 5]
```diff
@@ -203,9 +203,9 @@ export const resetPassword = email => dispatch => (
 		})
 );
 
-export const handleEmailPreferencesCheckboxChange = (name, checked) => dispatch => (
+export const subscribeToEmailList = name => dispatch => (
 	dispatch({
-		type: ACCOUNT_DATA_EMAIL_PREFERENCES_CHECKBOX_CHANGE,
-		payload: { name, checked },
```
[the remark this answers is not in the document]
Comment (benstrumeyer) [reply app/Account/AccountActions.js:209]: Good double check! I don't believe so, I made this const/action/reducer specifically for that checkbox

File: app/Account/AccountReducer.js (hunk 2 above)
Comment (benstrumeyer) [reply app/Account/AccountReducer.js:126]: I made this const to automatically subscribe people to sendgrid emails--the function is taken from a button in account-web that Teresa pointed me to. I believe `notify_promotions` and `notify_upgrade_updates` are different. `notify-promotions` are for displaying modals, and `notify_upgrade_updates` are for showing upgrade alerts when we push a new version. Can you double check me on this?

File: app/Account/AccountConstants.js (hunk 1 above)
Comment (benstrumeyer) [reply app/Account/AccountConstants.js:41]: We actually don't have the functionality in the original hub or the panel to toggle sendgrid emails on, so this is only used here

File: app/ghostery-browser-hub/Views/OnboardingViews/Step1_CreateAccountForm/Step1_CreateAccountFormContainer.jsx (hunk 3 above)
Comment (benstrumeyer) [reply app/ghostery-browser-hub/Views/OnboardingViews/Step1_CreateAccountForm/Step1_CreateAccountFormContainer.jsx:159]: See comment above

File: app/ghostery-browser-hub/Views/OnboardingViews/Step1_CreateAccountView/Step1_CreateAccountView.jsx (hunk 4 above)
Comment (benstrumeyer) [reply app/ghostery-browser-hub/Views/OnboardingViews/Step1_CreateAccountView/Step1_CreateAccountView.jsx:78]: Double checked with Teresa, we are cutting the arrow as well

File: app/Account/AccountReducer.js (hunk 2 above)
Comment (wlycdgr) [reply app/Account/AccountReducer.js:126]: You are right about `notify_promotions` and `notify_upgrade_updates`! We still may not need this action though if we now subscribe the user to emails automatically on the account server on account creation.

**Pull request merged**